helm/cluster-api/templates/core/mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
We need to change it to admissionregistration.k8s.io/v1, IIRC admissionregistration.k8s.io/v1beta1 was a long time ago deprecated.

Reply:
v1alpha3 (CAPI v0.3.x) is using this older version, it will be updated in v1beta1 (CAPI 1.x.x).

Reply:
Just saying that the admissionregistration.k8s.io/v1beta1 API version of MutatingWebhookConfiguration and ValidatingWebhookConfiguration is no longer served as of v1.22.

If you apply v1.22 in all installations we have a problem. I would not wait for upstream.

Reply:
This currently already running on our MCs, it's just deployed from another app here https://github.com/giantswarm/cluster-api-core-app/blob/v0.3.22-gs6/helm/cluster-api-core/templates/mutating-webhook.yaml#L2, so deploying this would not change anything.

These changes are just to update how we build and deploy CAPI and CRDs. v1beta1 (CAPI v1.0.2) with all new stuff would be deployed immediately after this.

Reply:
Thanks this will make sense 👍🏻